docs/oracle/oracle-price-example.md

```diff
@@ -65,7 +65,7 @@ predetermined interval, you need to sign the min (`0`) or the max (`$131,071`).
 Given the parameters we specified above, we are ready to create our digit decompisition event!
 
 ```
- ./bitcoin-s-cli createdigitdecompevent "bitcoin-s-price-example" 1612396800 2 17 "BTC/USD" 0
+ ./bitcoin-s-cli createnumericevent "bitcoin-s-price-example" 1612396800 2 17 "BTC/USD" 0
```

Comment:
These need to e updated, or even better move this into a "power user"/"advanced" section, and replace the existing section with a `createnumericevent` sample. 

I do think both examples should create teh same announcement, so the reader can compare and contrast the baked in assumptions the `createnumericevent` API makes (base2, inferred num digits,precision=0)

Reply:
Updated to explain both, lmk what you think